lib/shared/store.js

```diff
@@ -486,7 +487,42 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
   }
 };
 
-module.exports = _.merge(redux.createStore(storeReducer, DEFAULT_STATE), {
+const storeActionsAndDefaults = {
   Actions: ACTIONS,
   Defaults: DEFAULT_STATE
-});
+};
+
+// We're not in Electron
+if (typeof process === 'undefined') {
+  console.log('PLAIN REDUX');
+  module.exports = _.merge(redux.createStore(storeReducer, DEFAULT_STATE), storeActionsAndDefaults);
+
+// We are in the renderer process of Electron
+} else if (_.get(process, [ 'type' ]) === 'renderer') {
+  console.log('RENDERER REDUX');
```

Comment:
If this ever gets re-opened again, you'll need to remember to remove these `console.log`s ;-)